src/Components/WebAssembly/WebAssembly/src/HotReload/WebAssemblyHotReload.cs

```diff
@@ -27,7 +26,10 @@ public static class WebAssemblyHotReload
 
         internal static async Task InitializeAsync()
         {
-            if (!HotReloadEnvironment.Instance.IsHotReloadEnabled)
+            // Determine if we're running under a hot reload environment (e.g. dotnet-watch).
+            // It's insufficient to know it the app can be hot reloaded (HotReloadEnvironment.IsEnabled),
```

Comment:
`HotReloadEnvironment.IsEnabled` is getting removed in this PR, right? Does it make sense to refer to it here?

Reply:
```suggestion
            // It's insufficient to know it the app can be hot reloaded (HotReloadFeature.IsSupported),
```